R/dist.R

```diff
@@ -317,10 +317,15 @@ lognorm_dist_def <- function(mean, mean_sd,
                              to_log = FALSE) {
   
   if (to_log) {
-    mean <- log(mean)
```

Comment:
Thinking about this should this be happening after sampling? I think yes. Will check out and fix